### PR TITLE
[ENGDOCS-994] Update samples

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes
Updated samples page based on feedback.

### Related issues (optional)
ENGDOCS-994

### CODE REVIEW

File: samples/index.md [hunk 1]
```diff
@@ -1,18 +1,100 @@
...
+| Sample |
+| ------ |
+| [Gitea / PostgreSQL](https://github.com/docker/awesome-compose/tree/master/gitea-postgres){: target="_blank" rel="noopener" class="_"} |
+| [Nextcloud / PostgreSQL](https://github.com/docker/awesome-compose/tree/master/nextcloud-postgres){: target="_blank" rel="noopener" class="_"}
+| [Nextcloud / Redis / MariaDB](https://github.com/docker/awesome-compose/tree/master/nextcloud-redis-mariadb){: target="_blank" rel="noopener" class="_"} |
+| [Pi-hole / Cloudflared](https://github.com/docker/awesome-compose/tree/master/pihole-cloudflared-DoH){: target="_blank" rel="noopener" class="_"} |
+| [Prometheus / Grafana](https://github.com/docker/awesome-compose/tree/master/prometheus-grafana){: target="_blank" rel="noopener" class="_"} |
+| [Wordpress / MySQL](https://github.com/docker/awesome-compose/tree/master/wordpress-mysql){: target="_blank" rel="noopener" class="_"} |
+
+## Docker Samples on GitHub
```
Comment (usha-mandya) [inline samples/index.md:83]:
```suggestion
## Docker samples on GitHub
```

Comment (usha-mandya) [inline samples/index.md:83]: Wondering if it makes sense to move this to a new page to avoid a long scroll?

**Comment by usha-mandya** [comment]
@craig-osterhout Thanks for updating the Samples page. While we are at this, wondering if we can review the list of samples in the "Develop with Docker" section on the homepage and update the links there? Some links point to outdated samples.

image

File: samples/index.md (hunk 1 above)
Comment (craig-osterhout) [reply samples/index.md:83]: Thanks, @usha-mandya. I tested splitting it into an "Awesome Compose" and a "Docker Samples" page with some internal users and the feedback was grouping by those concepts didn't make sense to them and why do they need to go to multiple pages to see, for example, all the React samples.

I will try splitting it into separate pages based on the service. Of course, samples with multiple services will be duplicated among the pages, so I'm not fully confident that it's a good solution.

**Comment by usha-mandya** [comment]
Thanks @craig-osterhout. LGTM. Just a minor suggestion - can we open the external links in a new tab? Not a blocker though.

**Review by usha-mandya — APPROVED** [review]

**Review by jerae-duffin — APPROVED** [review]
LGTM! I can open all of the links in separate tabs

**Pull request merged**